### ISSUE TITLE
A request to remove an unsupported Stockalike Station Parts Redux mod.

### ISSUE THREAD

**Issue by linecrafter** [KSP-CKAN/NetKAN#6598]
Background
----------

**CKAN Version:**
1.25.1 

**KSP Version:**
1.3.1 

**Operating System:** 
Windows 10

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?** 
Yes

**Affected mods and mod versions:**
No idea what mods got affected. I'm still solving this issue.

Problem
-------

**What steps did you take in CKAN?**
Installed this mod with a check and pressed "Apply Changes" button

**What did you expect to happen?**
I wanted to have a great time playing with that mod

**What happened instead?**
I got an error message when loading my game (on screenshot). In the [thread](https://forum.kerbalspaceprogram.com/index.php?/topic/170211-142-stockalike-station-parts-redux-april-10/&) it says that this mod **does not** support (screenshot included) CKAN. 

**Screenshots:**
![default](https://user-images.githubusercontent.com/40280772/41443166-f2e380ce-7043-11e8-9083-9586f7934450.png)
![default](https://user-images.githubusercontent.com/40280772/41443200-21051e4a-7044-11e8-813a-a574012af7bc.png)

**Comment by politas** [comment]
That means that we need to determine whether the mod is correctly installed. 

- What version of B9PartSwitch does CKAN say you have installed?
- If you look in GameData/B9PartSwitch/ folder, you should see 6 files and a Plugins folder, with the Plugins folder containing a single DLL file
- Make sure you have "Stockalike Station Parts Expansion Redux" rather than "Stockalike Station Parts Expansion". The latter is retired, but both are available for KSP 1.3.1 in CKAN.
- Look in GameData/StationPartsExpansionRedux/ folder, you should see six folders and a single file called SSPXScienceDefs.cfg
- Open GameData/StationPartsExpansionRedux/Versioning/StationPartsExpansionRedux.version and check that the version reported there matches the version shown in CKAN.
- Search for 57E818B6-StationPartsExpansionRedux-1.0.2.zip file in your cache directory and compare the files under /GameData/StationPartsExpansionRedux/ in the zip with what you have in your GameData directory.
- If you still haven't found an issue, uninstall StationPartsExpansionRedux, NearFutureProps and B9PartSwitch in CKAN and install them manually from the zip file mentioned above and see if the game runs ok with that. If there's still a problem, you can report it in the forum thread without CKAN being involved.

**Comment by HebaruSan** [comment]
Probably not a CKAN related issue.

[Here's where SSPX defines that tank type](https://github.com/ChrisAdderley/StationPartsExpansionRedux/blob/434277978413cac0f0014bb4fb79d7fb8ec2bd40/GameData/StationPartsExpansionRedux/Patches/SSPXR-B9TankTypes.cfg#L361-L371):

```
B9_TANK_TYPE:NEEDS[CommunityResourcePack,!MKS]
{
	name = SSPXRocketParts
	tankMass = 0.0000
	tankCost = 0.0
	RESOURCE
	{
		name = RocketParts
		unitsPerVolume = 1
	}
}
```

The text between the brackets defines some conditions; if you have CommunityResourcePack installed and *NOT*  UKS, this tank type would be available, otherwise not. My guess is you had that situation previously, created a craft with this tank type, then removed CommunityResourcePack or installed UKS, and loaded your game, causing the mod to attempt to use this tank type when it's not defined. The solution would be to install CommunityResourcePack and/or uninstall UKS (whichever is the problem, or both).
